test/tests/kernels/2d_diffusion/tests

```diff
@@ -14,6 +14,15 @@
     scale_refine = 5
   [../]
 
+  [./testmatdiffusion]
+    type = 'Exodiff'
+    input = 'matdiffusion.i'
+    exodiff = 'matdiffusion_out.e'
+    requirement = 'MOOSE should test MatDiffusion Kernel'
```

Comment:
This requirement is too vague.  How about something like this:
'MOOSE shall provide a diffusion kernel that obtains the diffusion coefficient from a material property'